### PR TITLE
Clean up kwok init

### PR DESCRIPTION
**Pull request by sologgfun**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
The meaning of the parameter "Master" is unclear, and the descriptor uses the word "server".

#### Does this PR introduce a user-facing change?

```release-note
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kwok/cmd/root.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func NewCommand(ctx context.Context) *cobra.Command {
 	cmd.Flags().StringVar(&flags.Options.DisregardStatusWithAnnotationSelector, "disregard-status-with-annotation-selector", flags.Options.DisregardStatusWithAnnotationSelector, "All node/pod status excluding the ones that match the annotation selector will be watched and managed.")
 	cmd.Flags().StringVar(&flags.Options.DisregardStatusWithLabelSelector, "disregard-status-with-label-selector", flags.Options.DisregardStatusWithLabelSelector, "All node/pod status excluding the ones that match the label selector will be watched and managed.")
 	cmd.Flags().StringVar(&flags.Kubeconfig, "kubeconfig", flags.Kubeconfig, "Path to the kubeconfig file to use")
-	cmd.Flags().StringVar(&flags.Master, "master", flags.Master, "Server is the address of the kubernetes cluster")
+	cmd.Flags().StringVar(&flags.MasterUrl, "master-url", flags.MasterUrl, "MasterUrl is the address of the kubernetes cluster (https://hostname:port).")
```
Comment (wzshiming) [inline pkg/kwok/cmd/root.go:87]: This flag is the one that comes with the kube cli and should not be changed

**Review by wzshiming — CHANGES_REQUESTED** [review]

**Comment by sologgfun** [comment]
In addition, I have a small question.
```
// /pkg/kwon/cmd/root.go
RunE: func(cmd *cobra.Command, args []string) error {
	return runE(cmd.Context(), flags)
}

stagesData := config.FilterWithTypeFromContext[*internalversion.Stage](ctx)
```

The above code retrieves stagesData from ctx, but ctx is a new context, not the return parameter of `ctx, err := config.InitFlags(ctx, flagset)`
So where set the data to ctx？

File: pkg/kwok/cmd/root.go (hunk 1 above)
Comment (sologgfun) [reply pkg/kwok/cmd/root.go:87]: ok,but the description `Server is the address of the kubernetes cluster` looks like a parameter named `Server`.
And I find the same description field in k8s named `Server`

Comment (wzshiming) [reply pkg/kwok/cmd/root.go:87]: Sync to `kube-controller-manager`

https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/

```
--master string
The address of the Kubernetes API server (overrides any value in kubeconfig).
```

**Review by wzshiming — CHANGES_REQUESTED** [review]

File: pkg/kwok/cmd/root.go (hunk 1 above)
Comment (sologgfun) [reply pkg/kwok/cmd/root.go]: Okay, I have edited and pushed the new code.

File: pkg/kwok/cmd/root.go [hunk 2]
```diff
@@ -267,3 +249,29 @@ func filterStages(stages []*internalversion.Stage, apiGroup, kind string) []*int
 		return stage.Spec.ResourceRef.APIGroup == apiGroup && stage.Spec.ResourceRef.Kind == kind
 	})
 }
+
+func waitForReady(ctx context.Context, clientset kubernetes.Interface, logger *log.Logger) error {
```
Comment (wzshiming) [inline pkg/kwok/cmd/root.go:253]:
```suggestion
func waitForReady(ctx context.Context, clientset kubernetes.Interface) error {
	logger := log.FromContext(ctx)
```

Comment (sologgfun) [reply pkg/kwok/cmd/root.go:253]: ok, It`s looks better~

**Comment by sologgfun** [comment]
I have rewritten the redundant code. Please take another look. Thank you.

**Review by wzshiming — APPROVED** [review]

**Pull request merged**